### PR TITLE
Mex 359

### PR DESCRIPTION
**Pull request by dragos-rebegea**
## Reasoning
- Refactor LKMEX proposal for OldEnergy governance contract to use the same fragment as Energy, but having a static abi used to resolve fields.

### CODE REVIEW

**Review by gabrielmatei — DISMISSED** [review]

File: src/modules/governance/services/governance.abi.service.ts [hunk 1]
```diff
@@ -452,3 +453,93 @@ export class GovernanceEnergyAbiService
...
+        this.type = GovernanceType.OLD_ENERGY;
+    }
+
+
+    async energyFactoryAddress(scAddress: string): Promise<string> {
+        return Promise.resolve('');
+    }
+
+    async feeTokenId(scAddress: string): Promise<string> {
+        return Promise.resolve('LKMEX-aab910');
```
Comment (claudiulataretu) [inline src/modules/governance/services/governance.abi.service.ts:474]: I would not hardcode this value. Use the locked asset factory abi to get this value

File: src/modules/governance/services/governance.abi.service.ts [hunk 2]
```diff
@@ -452,3 +453,93 @@ export class GovernanceEnergyAbiService
...
+    async proposalRootHash(scAddress: string, proposalId: number): Promise<string> {
+        return Promise.resolve('');
+    }
+
+    async proposalStatus(scAddress: string, proposalId: number): Promise<GovernanceProposalStatus> {
+        return Promise.resolve(GovernanceProposalStatus.Succeeded);
+    }
+
+    async proposalVotes(scAddress: string, proposalId: number): Promise<ProposalVotes> {
+        return Promise.resolve(new ProposalVotes({
```
Comment (claudiulataretu) [inline src/modules/governance/services/governance.abi.service.ts:502]: extract them into entities folder

File: src/modules/governance/services/governance.abi.service.ts [hunk 3]
```diff
@@ -452,3 +453,93 @@ export class GovernanceEnergyAbiService
             .toPlainObject();
     }
 }
+
+@Injectable()
+export class GovernanceOldEnergyAbiService extends GovernanceEnergyAbiService {
```
Comment (claudiulataretu) [inline src/modules/governance/services/governance.abi.service.ts:458]: extract it into another service file

**Review by claudiulataretu — CHANGES_REQUESTED** [review]

File: src/modules/governance/services/governance.abi.service.ts (hunk 3 above)
Comment (dragos-rebegea) [reply src/modules/governance/services/governance.abi.service.ts:458]: extracted

File: src/modules/governance/services/governance.abi.service.ts (hunk 1 above)
Comment (dragos-rebegea) [reply src/modules/governance/services/governance.abi.service.ts:474]: done

File: src/modules/governance/services/governance.abi.service.ts (hunk 2 above)
Comment (dragos-rebegea) [reply src/modules/governance/services/governance.abi.service.ts:502]: done

**Review by claudiulataretu — APPROVED** [review]

**Review by gabrielmatei — APPROVED** [review]

**Pull request merged**